core/utils/poly-look/src/react/tabs/tabs.css

```diff
@@ -1,21 +1,25 @@
-.tab {
+.tabs .tab-buttons {
     margin: 0 auto;
     display: flex;
+    flex-direction: row;
     width: 100%;
     height: 40px;
 }
 
-.tab-button {
+.tabs .tab-buttons .tab-button {
     flex: 1 1 0px;
-    background-color: var(--color-background-light);
+    background-color: transparent;
```

Comment:
I would recommend you to use some color with opacity 0 (#00FFFFFF, for example) instead of `transparent` because transparent property usually appears as a dark grey color in Safari and iOS.